editors/emacs/Portfile

```diff
@@ -122,16 +123,16 @@ if {$subport eq "emacs-devel" || $subport eq "emacs-app-devel"} {
 
     # do not forget to check that configure hasn't introduce some suprises via
     # git diff [old]..[new] -- '**/configure.ac'
-    github.setup    emacs-mirror emacs f70a6ea0ea86ef461e40d20664a75a92d02679ea
-    epoch           5
-    version         20240804
+    github.setup    emacs-mirror emacs a06a7209028363a1bb2f727ffaecdf4d02296b2e
+    epoch           1
```

Comment:
the `epoch` field can never decrease, it should stay at `5` (see the [guide](https://guide.macports.org/chunked/reference.html))

Reply:
Whoops, I really should try find my MacPorts notes again.

Reply:
I'm going to bump it to `6` so it guaranteed invalidates prior versions.

Reply:
no, don't do that. You updates the version so there is absolutely no need to do so. The only reason to increase the "epoch" is when you want to downgrade the version or of the versioning has changed so that a newer version isn't recognized as newer by "vercomp".  

Please refer to the guide entry O linked.

Reply:
I was doing so just in-case as I had a brain fart and thought it also compared the commit hash (late and very tired). I did read the link you sent before acting hehe. I'll set it back to 5.